server/sonar-web/src/main/js/apps/quality-gates/components/Details.js

```diff
@@ -92,7 +89,7 @@ export default class Details extends React.PureComponent {
         router.replace(getQualityGatesUrl(organization && organization.key));
       }
     }).render();
-  }
+  };
 
   render() {
     const { qualityGate, edit, metrics } = this.props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "edit" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'edit' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AUnusedVariable)